leetcode/src/1147.c

```diff
@@ -0,0 +1,49 @@
+#define max(a,b) (((a)>(b))?(a):(b))
+
+bool substringsAreEqual(char* text, int firstIndex, int secondIndex, int length){

```

Comment:
We could probably rename the function's name here. What do you think?
```suggestion
bool equalSubstrings(char* text, int firstIndex, int secondIndex, int length){
```

Reply:
Yes, sure! good idea